sdk/formrecognizer/Azure.AI.FormRecognizer/src/Constants.cs

```diff
@@ -1,16 +1,35 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
+using System;
 
 namespace Azure.AI.FormRecognizer
 {
     internal static class Constants
     {
+        private const string AzurePublicCloud = "https://login.microsoftonline.com/";
+        private const string AzureChina = "https://login.microsoftonline.cn/";
+        private const string AzureGovernment = "https://login.microsoftonline.us/";
+
```

Comment:
These value are available in Azure.Identity [here](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/identity/Azure.Identity/src/AzureAuthorityHosts.cs)

Reply:
Would FR need to take a dependency on Azure.Identity to get those?  Should those go in Core instead?

Reply:
That's a good question. I was thinking, for some reason, that we'd already have the dependency. I think it may be valuable for this to move to Core as I've seen it replicated in a couple other places. 

@schaabs What do you think?

Reply:
I don't think these are necessary here, see my other comment regarding the reliance on `AZURE_AUTHORITY_HOST`.